client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionEventForm/useInteractionsForm.ts

```diff
@@ -1,132 +1,181 @@
+                            severity: Severity.LOW,
+                            workflow: 'Update Interaction',
+                            step: 'updated interaction successfully',
+                            user: userId,
+                            investigation: epidemiologyNumber
+                        });
+                        loadInteractions();
+                        closeEditDialog();
+                    }).catch((error) => {
+                    logger.error({
+                        service: Service.CLIENT,
+                        severity: Severity.LOW,
+                        workflow: 'Update Interaction',
+                        step: `got error from server: ${error}`,
+                        investigation: epidemiologyNumber,
+                        user: userId
+                    });
+                    handleFailedSave('לא ניתן היה לשמור את השינויים');
+                })
+            } else {
+                logger.info({
+                    service: Service.CLIENT,
+                    severity: Severity.LOW,
+                    workflow: 'Create Interaction',
+                    step: `launching create interaction request`,
+                    user: userId,
+                    investigation: epidemiologyNumber
+                });
+                axios.post('/intersections/createContactEvent', {
+                    ...interactionsDataToSave,
+                    [InteractionEventDialogFields.LOCATION_ADDRESS]: locationAddress,
+                    [InteractionEventDialogFields.INVESTIGATION_ID]: epidemiologyNumber,
+                    [InteractionEventDialogFields.CONTACTS]: interactionsDataToSave?.contacts?.map((contact: Contact) => ({
+                        ...contact,
+                        id: contact.idNumber
+                    }))
+                })
+                    .then(() => {
+                        logger.info({
+                            service: Service.CLIENT,
+                            severity: Severity.LOW,
+                            workflow: 'Create Interaction',
+                            step: 'created interaction successfully',
+                            user: userId,
+                            investigation: epidemiologyNumber
+                        });
+                        loadInteractions()
+                        closeNewDialog()
+                    })
+                    .catch((error) => {
+                        if (error.response.data === 'found duplicate ids') {
+                            logger.info({
+                                service: Service.CLIENT,
+                                severity: Severity.LOW,
+                                workflow: 'Create Interaction',
+                                step: 'Didnt save interactions due to duplicate ids',
+                                user: userId,
+                                investigation: epidemiologyNumber
+                            });
```

Comment:
Use custom swal

Reply:
In every swal

Reply:
in this swal im also adding a class for the container so i can't use the custom swal from the commons